locales/nl.json

```diff
@@ -25,25 +25,26 @@
       "basetemp": "Basistemperatuur",
       "schedule": "Huidige planning",
       "schedules": "Planningen",
+      "controlledOn": "Gecontroleerd aan",
       "operations": {
         "alwaysOn": "Altijd aan",
         "alwaysOff": "Altijd uit",
-        "controlledOn": "Voorkeur aan",
-        "controlledOff": "Aan tot Uit",
+        "controlOn": "Voorkeur aan",
+        "controledOff": "Aan tot Uit",
```

Comment:
Note that the reference value for the translation must not be changed.
You can run the ./validate.sh script in order to check that all the references are working.
I'll fix it ;-)

Reply:
Of course. 
I can't understand why that was changed?
I hate the PR system even more now 😅😅
And I missed it obviously while checking the changes.🤔